src/mon/MgrStatMonitor.cc

```diff
@@ -69,70 +69,64 @@ void MgrStatMonitor::create_initial()
 void MgrStatMonitor::calc_pool_availability()
 {
   dout(20) << __func__ << dendl;
-  auto pool_avail_end = pool_availability.end();
-  for (const auto& i : digest.pool_pg_unavailable_map) {
-    const auto& poolid = i.first;
-    if (pool_availability.find(poolid) == pool_avail_end){
-      // New Pool so we add.
-      pool_availability.insert({poolid, PoolAvailability()});
-      dout(20) << __func__ << "Adding pool: " << poolid << dendl;
+
+  // Add available pools
+  for (const auto& [poolid, _] : digest.pool_pg_unavailable_map) {
```

Comment:
as we are still C++-20: don't you get 'unused' warnings for the '_'?
If so - maybe mark it as [[maybe_unused]]] ?